locales/locales.php

```diff
@@ -2050,6 +2050,15 @@ public function __construct() {
 		$rhg->slug = 'rhg';
 		$rhg->nplurals = 1;
 		$rhg->plural_expression = '0';
+		
+		$rif = new GP_Locale();
+		$rif->english_name = 'Tarifit';
+		$rif->native_name = 'Tarifiyt';
+		$rif->lang_code_iso_639_3 = 'rif';
+		$rif->country_code = 'ma';
+		$rif->slug = 'rif';
+		$rif->nplurals = 2;
+		$rif->plural_expression = 'n > 1';
```

Comment:
```suggestion
```
The plural_expression is `n != 1` (the default one), as you can check [here](https://translations.launchpad.net/+languages/rif), so we can remove it.